### DISCUSSION TITLE
Tortoise development

### DISCUSSION THREAD

**Discussion by evstratbg** [tortoise/tortoise-orm#1375] [Ideas]
Hello folks, @grigi @abondar @long2ice 

I am reaching out to discuss the Tortoise-ORM Python framework with you. While it appears that development has come to a standstill, my company has found it to be an invaluable tool and continues to use it in our production environment. We have several features that we would like to contribute to the project, but unfortunately, progress on these features has stalled due to a lack of maintainer activity.

With that in mind, I would like to inquire about the possibility of obtaining maintainer permissions for the Tortoise-ORM GitHub repository and PyPI. By taking on this role, my team would be able to actively contribute to the development of the framework and ensure that it remains a relevant and useful tool for the Python community. 

Thank you for your consideration, and I look forward to hearing back from you soon.

**Comment by vlakius** [comment]
I hope this will be taken seriously. In my company we use tortoise orm in production for some products and there would be many aspects to improve
